benchmark/src/ttft.ts

```diff
@@ -0,0 +1,46 @@
+import cp, { StdioOptions } from "child_process";
+import fs from "fs";
+
+import { setup, TURBO_BIN, DEFAULT_EXEC_OPTS, REPO_PATH } from "./helpers";
+
+(async function () {
+  cp.execSync(`${TURBO_BIN} --version`, { stdio: "inherit" });
+
+  console.log("setup");
+  setup();
+
+  if (!fs.existsSync(TURBO_BIN)) {
+    throw new Error("No turbo binary found");
+  }
+
+  console.log("running ttft", {
+    cwd: process.cwd(),
+    bin: TURBO_BIN,
+  });
+
+  // Path to profile.json is ../profile.json because we are in
+  // benchmark/large-monorepo (i.e. REPO_PATH) when this runs
+  const opts = {
+    ...DEFAULT_EXEC_OPTS,
+    stdio: "inherit" as StdioOptions,
+    env: {
+      TURBO_LOG_VERBOSITY: "trace",
+      EXPERIMENTAL_RUST_CODEPATH: "true",
+    },
+  };
+  console.log("Executing turbo build in child process with opts", opts);
+
+  try {
+    cp.execSync(
+      `${TURBO_BIN} run build --skip-infer --force --dry --profile ../profile.json`,
+      opts
+    );
+  } catch (e) {
+    // not sure why this is erroring
```

Comment:
This seems suspicious. Maybe just drop the `try ... catch` and let it error if it's going to error?

Reply:
removing the pipe did the trick. probably the same issue as we got from a customer recently where large output fills up the max buffer? I tried increasing it a bit (maybe not enough) but that didn't fix it, so just swallowing the output. we don't need it

Reply:
I brought it back because it still failed in Windows for the same reason: https://github.com/vercel/turbo/actions/runs/6542776699/job/17766330242

I'd like to merge this and then come back to this after we're reduced the amount of output required for tracing